.github/workflows/publish-docker.yml

```diff
@@ -0,0 +1,43 @@
+name: 'Publish Docker'
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to publish Docker image for (scip-ruby-vM.N.P)'
+  workflow_run:
+    workflows: ['Release']
+    types:
+      - completed
+
+jobs:
+  release-image:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          if [ -z "$TAG" ]; then
+            # From https://stackoverflow.com/questions/10649814/get-last-git-tag-from-a-remote-repo-without-cloning
+            TAG="$(git -c 'versionsort.suffix=-' \
+              ls-remote --exit-code --refs --sort='version:refname' --tags https://github.com/sourcegraph/scip-ruby.git 'scip-ruby-v*' \
+              | tail --lines=1 \
+              | cut --delimiter='/' --fields=3)"
+          fi
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
+        env:
+          TAG: ${{ inputs.tag }}
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ env.TAG }}
```

Comment:
This scheme doesn't let us generate Docker images for older releases (I'd like to make one for 0.3.0) but maybe that's fine.